test/files/jvm/serialization.check

```diff
@@ -97,12 +97,12 @@ x = Queue(a, b, c)
 y = Queue(a, b, c)
 x equals y: true, y equals x: true
 
-x = Range(0, 1, 2, 3, 4, 5, 6, 7, 8, 9)
-y = Range(0, 1, 2, 3, 4, 5, 6, 7, 8, 9)
+x = 0 until 10 by 1
```

Comment:
If I was browsing a log file `Range(0 until 10 by 1)` would be of greater benefit.